kythe/cxx/indexer/cxx/marked_source.cc

```diff
@@ -551,6 +566,23 @@ std::string GetDeclName(const clang::LangOptions& lang_options,
   return "";
 }
 
+void CleanMarkedSource(MarkedSource* to_clean) {
```

Comment:
This should be in the anonymous namespace above (or marked `static`). So should `GetDeclName`, but that's pre-existing.